cmd/snap-preseed/reset.go

```diff
@@ -98,5 +99,45 @@ func resetPreseededChroot(preseedChroot string) error {
 		}
 	}
 
+	// bash-completion symlinks; note there are symlinks that point at
+	// completer, and symlinks that point at the completer symlinks.
+	// e.g.
+	// lxd.lxc -> /snap/core/current/usr/lib/snapd/complete.sh
+	// lxc -> lxd.lxc
+	files, err := ioutil.ReadDir(filepath.Join(preseedChroot, dirs.CompletersDir))
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("error reading %s: %v", dirs.CompletersDir, err)
+	}
+	completeShSymlinks := make(map[string]string)
+	var otherSymlinks []string
+
+	// pass 1: find all symlinks pointing at complete.sh
+	for _, fileInfo := range files {
+		if fileInfo.Mode()&os.ModeSymlink == 0 {
+			continue
+		}
+		fullPath := filepath.Join(preseedChroot, dirs.CompletersDir, fileInfo.Name())
+		if dirs.IsCompleteShSymlink(fullPath) {
```

Comment:
we probably know more things in the other places that use that helper, but looking at the helper there it seems too weak for using on all files in completion, afaict we should fix that helper to at least check for snapd/complete.sh